### PR TITLE
Request to merge in 1.0.3 changes

### PR DESCRIPTION
**Pull request by fortijames**
_Do NOT Release before the v1.0.3 Framework is released_
https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/pull/16

**1.0.3** 
 - New Profile that shows an example of a custom script. 
 - Modify the script `Create Loopback - lo0 - DeviceDB` to include `set ip {{devmeta.loopback0_ip}}/32`.

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -26,33 +26,38 @@
...
     "contents": {
+        "playbooks": [
+            {
+                "name": "10 - SP - FortiManager ZTP Flow - FortiManager(FMG)-CustomPlaybooks"
+            }
+        ],
         "recordSets": [
             {
-                "name": "Attachments",
+                "name": null,
```
Comment (cs-rohit-khune) [inline info.json]: "name" value should be *Attachments*

File: info.json [hunk 2]
```diff
@@ -26,33 +26,38 @@
...
                 "apiName": "attachments",
                 "count": 3
             },
             {
-                "name": "Metafield Templates",
+                "name": null,
```
Comment (cs-rohit-khune) [inline info.json]: "name" value should be*Metafield Templates*

File: info.json [hunk 3]
```diff
@@ -26,33 +26,38 @@
...
                 "apiName": "metafield_templates",
                 "count": 10
             },
             {
-                "name": "Scripts",
+                "name": null,
```
Comment (cs-rohit-khune) [inline info.json]: "name" value should be *Scripts*

File: info.json [hunk 4]
```diff
@@ -26,33 +26,38 @@
...
                 "apiName": "scripts",
-                "count": 5
+                "count": 7
             },
             {
-                "name": "ZTP Profiles",
+                "name": null,
```
Comment (cs-rohit-khune) [inline info.json]: "name" value should be *ZTP Profiles*

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

**Comment by fortijames** [comment]
Changed the recordSets in info.json. Please let me know if there are any other pending changes preventing a merge.

**Pull request merged**